tutorials/Python/Python-skills/using-jupyter-notebooks/JupEnd-CulmActivity/JupEnd-CulmActivity.md

```diff
@@ -16,125 +17,117 @@ urlTitle: jupyter-python-cul-activity
-**NEONScience/DI-NEON-participants** GitHub repo as a **pull request**
-by 11:59pm on 16 June 2017.
+This will be accomplished by:
+  *Create a new  Jupyter kernel
+  *Download a GEOTIFF file
+  *Import file onto Jupyter Notebooks
+  *Check the raster size
+
 
-</div>
 
 ## Assignment: Open a Tiff File in Jupyter Notebook 
 
-Download the NEON GeoTiFF file of the 
-<a href="https://neondata.sharefile.com/d-s9297db4154a4dceb"> digital terrain model (DTM) </a> 
-from San Joaquin Experimental Range collected in 2017. 
-. 
-Open this file in Jupyter Notebooks, determine the size of the raster, and 
-(optional extension) plot the raster. Add in both code chunks and text (markdown) chunks
-to fully explain what is done. 
-When finished, submit the .ipynb file to the 
-**NEONScience/DI-NEON-participants** GitHub repo 
 
-## Detailed Directions
 
 ### Set up Environment 
 
 First, we will set up the environment as you would need for each of the live 
 coding sections of the Data Institute. The following directions are copied over
 from the Data Institute Set up Materials.
 
-Note, we've had reports from some individuals that Python 3.6 was able to use 
-GDAL, however, we have others who are not able to use Python 3.6 and still have 
-to use Python 3.4.  
-
 In your terminal application, navigate to the directory (`cd`) that where you
 want the Jupyter Notebooks to be saved (or where they already exist). 
 
-We need to create a new Jupyter kernel for the Python 3.4 conda environment 
-(py34) that Jupyter Notebooks will use. 
+We need to create a new Jupyter kernel for the Python 3.8 conda environment 
+(py38) that Jupyter Notebooks will use. 
 
 In your Command Prompt/Terminal, type: 
 
-`python -m ipykernel install --user --name py34 --display-name "Python 3.4 NEON-RSDI"`
+`python -m ipykernel install --user --name py34 --display-name "Python 3.8 NEON-RSDI"`
 
 In your Command Prompt/Terminal, navigate to the directory (`cd`) that you 
 created last week in the GitHub materials. This is where the Jupyter Notebook 
 will be saved and the easiest way to access existing notebooks. 
 
-Open Jupyter Notebook with 
+###Open Jupyter Notebook
+Open Jupyter Notebook by typing into a command terminal:
 
 `jupyter notebook`
 
 Once the notebook is open, check which version of Python you are in. 
 
-	 # Check what version of Python.  Should be 3.4. 
+	 # Check what version of Python.  Should be 3.8. 
 	 import sys
 	 sys.version
 
 To ensure that the correct kernel will operate, navigate to **Kernel** in the menu, 
-select **Restart/ClearOutlook**. 
+select **Kernel/Restart Kernel And Clear All Outputs**. 
 
  <figure>
 	<a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/pre-institute-content/pre-institute3-jupPy/jupPy-kernel.png">
 	<img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/pre-institute-content/pre-institute3-jupPy/jupPy-kernel.png"></a>
```

Comment:
please add 'alt text' to this img (even though it may be redundant with the figure caption, it's best to include anyways)